src/components/tooltips/line-popover.js

```diff
@@ -0,0 +1,290 @@
+                                                    lineInfo.voltageLevelId1
+                                                )}
+                                            </TableCell>
+                                            <TableCell>
+                                                {checkValue(
+                                                    lineInfo.voltageLevelId2
+                                                )}
+                                            </TableCell>
+                                        </TableRow>
+                                    </TableHead>
+                                    <TableBody>
+                                        <TableRow>
+                                            <TableCell>
+                                                {intl.formatMessage({
+                                                    id: 'I_(A)',
+                                                })}
+                                            </TableCell>
+                                            <TableCell>
+                                                {loadFlowStatus ===
+                                                RunningStatus.SUCCEED
+                                                    ? checkValue(
+                                                          Math.round(
+                                                              lineInfo.i1
+                                                          )
+                                                      )
+                                                    : null}
+                                            </TableCell>
+                                            <TableCell>
+                                                {loadFlowStatus ===
+                                                RunningStatus.SUCCEED
+                                                    ? checkValue(
+                                                          Math.round(
+                                                              lineInfo.i2
+                                                          )
+                                                      )
+                                                    : null}
+                                            </TableCell>
+                                        </TableRow>
+                                    </TableBody>
+                                </Table>
+                            </TableContainer>
+                        </Grid>
+
+                        <Grid item>
+                            <TableContainer component={Paper}>
+                                <Table size={'small'}>
+                                    <TableHead>
+                                        <TableRow>
+                                            <TableCell>
+                                                {intl.formatMessage({
+                                                    id: 'LimitType',
```

Comment:
We need :
Seuil (fr) = Limit name (en) 

A new key is needed.

Reply:
fixed